app/frontend/src/views/admin/Root.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <v-container>
     <BaseSecure>
-      <SubmissionsTable class="my-8" />
+      <h1>Submissions</h1>
```

Comment:
IIRC Mina usually mentioned that there needs to be some degree of margin around the header - if we're putting in a new h1 here, make sure it has at least an my-6 or larger.